src/components/pages/accounts/NetWorthHistogram.tsx

```diff
@@ -0,0 +1,217 @@
+import React from 'react';
+import { DateTime, Interval } from 'luxon';
+
+import Chart from '@/components/charts/Chart';
+import type { AccountsTree } from '@/types/accounts';
+import { moneyToString } from '@/helpers/number';
+
+export type NetWorthHistogramProps = {
+  startDate?: DateTime,
+  tree: AccountsTree,
+};
+
+export default function NetWorthHistogram({
+  startDate,
+  tree,
+}: NetWorthHistogramProps): JSX.Element {
+  const interval = Interval.fromDateTimes(
+    startDate || DateTime.now(),
+    DateTime.now(),
+  );
+  const dates = interval.splitBy({ month: 1 }).map((d) => (d.start as DateTime).plus({ month: 1 }));
```

Comment:
We should replicate this logic to all the other charts rather than hardcoding months and use time series... would make the series more manageable.